arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -1645,6 +1645,14 @@
 			</dict>
 			<dict>
 				<key>descriptionText</key>
+				<string>Create graphics for utility associations in a utility network.</string>
+				<key>storyboardName</key>
+				<string>DisplayUtilityAssociations</string>
+				<key>displayName</key>
+				<string>Display utility associations</string>
+			</dict>
+			<dict>
+                <key>descriptionText</key>
```

Comment:
The indentation is not correct here. Simply reopening the file inside the project and saving it (Cmd + S) should fix the problem.